dev/breeze/doc/ci/06_diagrams.md

```diff
@@ -1,466 +0,0 @@
-<!--
```

Comment:
There is only one workflow (two types - PR and canary (main/v*-test) ones, so we can remove the diagrams.